ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -283,20 +289,20 @@ class TransactionDetailsScreen extends React.Component<Props> {
               <H5 style={styles.value}>{totalAmount}</H5>
             )}
             {this.labelValueRow(I18n.t("wallet.payAmount"), amount)}
-            {this.labelValueRow(
-              <Text>
-                <Text note={true}>{`${I18n.t("wallet.transactionFee")} `}</Text>
-                <Text
-                  note={true}
-                  bold={true}
-                  style={styles.whyLink}
-                  onPress={this.showHelp}
-                >
-                  {I18n.t("wallet.why")}
-                </Text>
-              </Text>,
-              fee
-            )}
+            <TouchableWithoutOpacity onPress={this.showHelp}>
+              {this.labelValueRow(
+                <Text>
+                  <Text note={true}>{`${I18n.t(
+                    "wallet.transactionFee"
+                  )} `}</Text>
+
+                  <Text style={styles.linkStyle} note={true}>
```

Comment:
I guess this style is not needed. You can leave it as it was
```
<Text
  note={true}
  bold={true}
  style={styles.whyLink}
  onPress={this.showHelp}
>
```

Reply:
onPress props remains on TouchableDeafultOpacity